imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -370,6 +366,8 @@
    },
    "outputs": [],
    "source": [
+    "# tensorflow==1.14.0 to have same versions in all environments\n",
```

Comment:
"all environments": can you please clarify what do you mean by this? What are the other environments that are relevant here?

Reply:
done